lasagne/layers/base.py

```diff
@@ -262,30 +234,17 @@ def __init__(self, incomings, name=None):
                 an optional name to attach to this layer
         """
         self.input_shapes = [incoming if isinstance(incoming, tuple)
-                             else incoming.get_output_shape()
+                             else incoming.output_shape
                              for incoming in incomings]
         self.input_layers = [None if isinstance(incoming, tuple)
                              else incoming
                              for incoming in incomings]
         self.name = name
 
-    def get_output_shape(self):
+    @Layer.output_shape.getter
```

Comment:
Just curious: why is this necessary? Can't properties be overridden or something? (I would have expected `@property`.)